src/Game/Maps.elm

```diff
@@ -187,6 +189,18 @@ currentAreaMap (A model) =
                 Dict.empty
 
 
+{-| Get the width and height of the map for the current area
+-}
+currentAreaSize: Maps -> Size
```

Comment:
It's on ln:42, `import Window exposing (Size)`, it's actually a
```
type alias Size =
  { width : Int
  , height : Int
  }
```

which I think makes it unsuitable for this. I'd rather `Vector.elm` had a `type alias Dimension = Vector` and this used Dimension. Let's not tie Map to Window.